CMakeLists.txt

```diff
@@ -78,7 +78,8 @@ FetchContent_MakeAvailable(repo-common repo-core repo-backend)
 FetchContent_Declare(
   pybind11
   GIT_REPOSITORY "https://github.com/pybind/pybind11"
-  GIT_TAG "v2.10"
+  # COMMIT ID for v2.10.0
```

Comment:
Thanks for the detailed explanation.

I'm happy to pin to commit hash or v2.10.0 to avoid unintended patch version updates for now.

I think we should include a slightly more detailed comment about the locale issue so it's clear as a possible consequence when updating the version in the future.

Reply:
Moved comment suggestion into test: https://github.com/triton-inference-server/server/pull/5962